.github/workflows/test_linux_build_on_windows.yml

```diff
@@ -53,6 +56,8 @@ jobs:
       - name: List files
         run: ls
       - name: Install project-keeper
-        run: mvn --batch-mode install:install-file -Dfile="project-keeper.jar" -D pomFile="pom.xml"
+        run: mvn install:install-file -Dfile="project-keeper.jar" `
+            --batch-mode `
+            -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
```

Comment:
Here you have back ticks as separator, I think it is a typo.

Reply:
This runs on powershell which requires backticks as line continuation chars.